toml.abnf

```diff
@@ -84,9 +84,10 @@ ml-basic-string = ml-basic-string-delim ml-basic-body ml-basic-string-delim
 
 ml-basic-string-delim = 3quotation-mark
 
-ml-basic-body = *( ml-basic-char / newline / ( escape ws newline ) )
-ml-basic-char = ml-basic-unescaped / escaped
+ml-basic-body = *( ml-basic-char / wschar / newline / ml-basic-escaped-nl )
+ml-basic-char = ( ml-basic-unescaped / escaped )
```

Comment:
The parentheses surrounding the role are unnecessary and should be dropped (reverting to the one rule).